src/utils/AnimationUtils.js

```diff
@@ -31,7 +31,33 @@
 define(function (require, exports, module) {
     "use strict";
     
-    var Async   = require("utils/Async");
+    var _     = require("thirdparty/lodash"),
+        Async = require("utils/Async");
+    
+    /**
+     * @private
+     * Detect the browser's supported transitionend event.
+     * @return {string} The supported transitionend event name.
+     */
+    function _detectTransitionEvent() {
+        var event, el = document.createElement("fakeelement");
+        
+        var transitions = {
+            "OTransition"     : "oTransitionEnd",
+            "MozTransition"   : "transitionend",
+            "WebkitTransition": "webkitTransitionEnd",
+            "transition"      : "transitionend",
```

Comment:
Upon inspection, I found that placing the unprefixed version first would make `webkitTransitionEnd` the returned value. That stems from being unable to break `_.forEach`. Placing them in this order fixes that and (on Windows), correctly returns `transitionend`.